src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" data-qa="big">
+      <fieldset class="block big">
```

Comment:
`fieldset` is redundant, you don't need it here

Reply:
Shall I take out both fieldsets?